libobs/obs-video-gpu-encode.c

```diff
@@ -125,10 +134,27 @@ static void *gpu_encode_thread(struct obs_core_video_mix *video)
 			else
 				next_key++;
 
-			success = encoder->info.encode_texture(
-				encoder->context.data, tf.handle,
-				encoder->cur_pts, lock_key, &next_key, &pkt,
-				&received);
+			profile_start(gpu_encode_frame_name);
+			if (encoder->info.encode_texture2) {
+				struct encoder_texture tex = {0};
+
+				obs_encoder_get_video_info(encoder, &tex.info);
```

Comment:
As noted above we don't really need the video info in the texture metadata so this can be removed.